Modules/mathmodule.c

```diff
@@ -2347,7 +2347,7 @@ math_fmod_impl(PyObject *module, double x, double y)
     if (isinf(y) && isfinite(x))
         return PyFloat_FromDouble(x);
     errno = 0;
-    r = fmod(x, y);
+    r = m_fmod(x, y);
```

Comment:
I don't know mathmodule.c, so maybe my question is stupid. But why not calling directly _Py_fmod()?